### PR TITLE
Bump flannel and calico for canal

### PR DESCRIPTION
**Pull request by raulcabello**

### CODE REVIEW

File: packages/rke2-canal/charts/values.yaml [hunk 1]
```diff
@@ -25,15 +25,15 @@ calico:
   # CNI installation image.
   cniImage:
     repository: rancher/hardened-calico
-    tag: v3.20.1-build20211006
+    tag: v3.20.1-build20211030
```
Comment (dweomer) [inline packages/rke2-canal/charts/values.yaml:28]: @raulcabello not https://github.com/rancher/image-build-calico/releases/tag/v3.20.2-build20211115 ?

**Comment by raulcabello** [comment]
@dweomer I updated to the latest version

**Review by dweomer — APPROVED** [review]

**Review by brandond — APPROVED** [review]

**Comment by dweomer** [comment]
@raulcabello looks like there are some conflicts

**Comment by brandond** [comment]
Superseded by https://github.com/rancher/rke2-charts/pull/190

**Pull request closed without merging**